pkg/initializer/initializer.go

```diff
@@ -35,6 +35,11 @@ import (
 	"k8s.io/client-go/util/retry"
 )
 
+const (
+	// retrySteps is the no. of steps used for exponential backoff to add a learner.
+	retrySteps = 6
```

Comment:
this is too generic and today it is only used to add a learner. Can we rename this to `addLearnerAttempts`. The description should just say `addLearnerAttempts are the total number of attempts that will be made to add a learner` - no need to mention exponential backoff as the number of attempts has nothing to do with how you wait between 2 attempts.